app/api/crud.py

```diff
@@ -192,9 +192,52 @@ async def get_terms(data_element_URI: str):
 
     results_dict = {
         data_element_URI: [
-            result["termURL"]["value"]
+            util.replace_namespace_uri(result["termURL"]["value"])
             for result in results["results"]["bindings"]
         ]
     }
 
     return results_dict
+
+
+async def get_controlled_term_attributes():
```

Comment:
Is this function going to be refactored as well along with the `get`?

Reply:
Oui - the function itself will likely stick around, but the `httpx.post` call will be refactored out since the arguments are largely shared across all the `get_` crud functions.